reco_utils/dataset/movielens.py

```diff
@@ -128,50 +135,50 @@ def item_has_header(self):
 ERROR_MOVIE_LENS_SIZE = "Invalid data size. Should be one of {100k, 1m, 10m, or 20m}"
 ERROR_LOCAL_CACHE_PATH = """Local cache path only accepts a zip file path:
     use/something/like_this.zip"""
+ERROR_NO_HEADER = "No header (schema) information"
 
 
 def load_pandas_df(
     size="100k",
-    header=(
-        DEFAULT_USER_COL,
-        DEFAULT_ITEM_COL,
-        DEFAULT_RATING_COL,
-        DEFAULT_TIMESTAMP_COL
-    ),
-    local_cache_path="ml.zip",
+    header=DEFAULT_HEADER,
+    local_cache_path=None,
     title_col=None,
```

Comment:
The column names are `col_user` for example in some other utilities. Do we want to make them consistent?

Reply:
Good suggestion! I personally prefer to name a variable as we speak. e.g. "column name" -> `col_name` and "user column" -> `user_col`. What do you guys think?

Reply:
I see. No strong personal preference here lol. Maybe later we can change these all together.